core/src/spotify_id.rs

```diff
@@ -84,14 +36,6 @@ impl SpotifyId {
     const SIZE_BASE16: usize = 32;
     const SIZE_BASE62: usize = 22;
 
-    /// Returns whether this `SpotifyId` is for a playable audio item, if known.
-    pub fn is_playable(&self) -> bool {
```

Comment:
And that some methods were removed from `SpotifyId` and moved to `SpotifyUri` is also probably note worthy for the Changelog. This can probably be summarized as "some methods where moved to `SpotifyUri` tho.